cyraft/election_watchdog.py

```diff
@@ -43,11 +47,15 @@ async def _unittest_election_watchdog() -> None:
     watchdog_node = ElectionWatchdog()
     asyncio.create_task(watchdog_node.run())
 
-    await asyncio.sleep(ELECTION_TIMEOUT - 1)  # Election timeout should not be reached here
+    await asyncio.sleep(
+        ELECTION_TIMEOUT - 1
+    )  # Election timeout should not be reached here
 
     watchdog_node.reset_election_timeout()  # Reset election timeout
 
-    await asyncio.sleep(1)  # Election timeout should be reached here, but it should be reset above
+    await asyncio.sleep(
+        1
+    )  # Election timeout should be reached here, but it should be reset above
```

Comment:
This will be re-formatted after adjusting line length.

Reply:
[Fixed](https://github.com/OpenCyphal-Garage/cyraft/pull/11/commits/e9cd8165d7ba66c545d75d3037f5b044045cc01d)